Example/ios/ScreensExample.xcodeproj/project.pbxproj

```diff
@@ -170,7 +170,6 @@
 				F3E913972B31574447799D3B /* Pods-ScreensExample.debug.xcconfig */,
 				A73D6623F5213E78782B8867 /* Pods-ScreensExample.release.xcconfig */,
 			);
-			name = Pods;
```

Comment:
Why is it deleted?

Reply:
I don't know. I edited this file in xcode, so I think it was automatically done